robosuite/environments/base.py

```diff
@@ -195,15 +195,19 @@ def step(self, action):
             raise ValueError("executing action in terminated episode")
 
         self.timestep += 1
-        self._pre_action(action)
-        end_time = self.cur_time + self.control_timestep
-        while self.cur_time < end_time:
+        policy_step = True
```

Comment:
comments for what the `policy_step` variable does would be useful - explain that the env.step frequency is different from the number of simulation steps since a controller will output torques multiple times for a single policy output

Reply:
@cremebrule Please address this comment

Reply:
Sorry -- missed this comment. done!